app/views/permissions/_form.html.erb

```diff
@@ -2,70 +2,126 @@
+        <%= radio_button_tag 'collection_option', 'all', @permission.empty? ? true : @permission.fetch('catalog_item_identity', {}).fetch('collection_identifier', {}).fetch('entry_titles', []).empty? %>
+        All Collections
+      <% end %>
+      <p class="form-description">If any access constraints are applied it will filter all available collections.</p>
     </div>
-    <div class="col-6 min-max-col">
-      <%= label_tag 'collection_access_value[max_value]', 'Maximum Access Constraint Value' %>
-      <%= text_field_tag 'collection_access_value[max_value]', @collection_access_value['max_value'], class: 'full-width min-max-value collection-access-value-group' %>
+
+    <div class="col-6">
+      <%= label_tag 'collection_option_selected' do %>
+        <%= radio_button_tag 'collection_option', 'selected', @permission.fetch('catalog_item_identity', {}).fetch('collection_identifier', {}).fetch('entry_titles', []).any? %>
+        Selected Collections
+      <% end %>
+      <p class="form-description">If any access constraints are applied it will filter the selected collections further.</p>
     </div>
   </div>
-  <div class="row checkbox-group">
-    <%= label_tag 'collection_access_value[include_undefined_value]' do %>
-      <%= check_box_tag 'collection_access_value[include_undefined_value]', true, @collection_access_value['include_undefined_value'] %>
-      Include Undefined
-    <% end %>
-  </div>
-</fieldset>
 
-<fieldset>
-  <%= label_tag 'granule_options', 'Granules', class: 'required eui-required-o' %>
-  <%= select_tag 'granule_options', options_for_select(PermissionsHelper::GranulesOptions, @granule_options), class: 'full-width', prompt: '-- Select an Option --' %>
+  <!-- Hidden fields that represent the underlying array of selected collections -->
+  <% Array.wrap(@permission.fetch('catalog_item_identity', {}).fetch('collection_identifier', {}).fetch('entry_titles', [])).each do |collection| %>
+  <%= hidden_field_tag('collection_selections[]', collection, class: 'selected-collection') %>
+  <% end %>
+
+  <!-- Hidden fields that represent the underlying array of selected collections that the user doesn't have access to -->
+  <% Array.wrap(unauthorized_permission_collections(@permission)).each do |entry_title| %>
+    <%= hidden_field_tag('hidden_collections[]', entry_title, class: 'hidden-collection') %>
+  <% end %>
+  <div id="chooser-widget"></div>
 </fieldset>
 
-<fieldset id="granule_constraint_values" class="<%= hide_access_constraint_values?(@granule_options) %>">
-  <h3 class="space-bot">Granules Access Constraint Filter</h3>
-  <div class="row min-max-row">
-    <div class="col-6 min-max-col">
-      <%= label_tag 'granule_access_value[min_value]', 'Minimum Access Constraint Value' %>
-      <%= text_field_tag 'granule_access_value[min_value]', @granule_access_value['min_value'], class: 'full-width min-max-value granule-access-value-group' %>
+<fieldset id="granule_constraint_values">
```

Comment:
I feel like this id should be modified since it includes the filters for collections